packages/frontend/src/app/heat-map/heat-map.component.spec.ts

```diff
@@ -54,6 +54,7 @@ describe('HeatMapComponent', () => {
     fixture.autoDetectChanges(true);
     component.config.weeksToDisplay = 2;
     component.config.daysToDisplay = 7;
+    component.init('', '');
```

Comment:
what do  these two empty strings do?

Reply:
If they were used in those tests, that line would be `component.init('repoName', 'orgName')`.